schema/v5.0/CVE_JSON_5.0_schema.json

```diff
@@ -561,8 +561,9 @@
                 "references"
             ],
             "patternProperties": {
-                "^x_[^.]*$": {}
+                "^x_[^.$]*$": {}
```

Comment:
Restricting `$` is overly restrictive. The problem with `$` only exists if the property name starts with `$`. The pattern already requires that properties star with `x_`, so no further restriction is needed.

Need to remove the `$` restriction in this and the following patterns per the discussion during the 6/22/2023 QWG call.